cli_helpers/utils.py

```diff
@@ -55,7 +55,7 @@ def unique_items(seq):
     return [x for x in seq if not (x in seen or seen.add(x))]
 
 
-_ansi_re = re.compile('\033\[((?:\d|;)*)([a-zA-Z])')
+_ansi_re = re.compile('\033\\[((?:\\d|;)*)([a-zA-Z])')
```

Comment:
Would this be better served by using a raw-string instead of this double escaping?

Reply:
Yes, I guess using raw strings helps not sure why the tool used escapes again.

Reply:
Using raw string does pose a difference where escaping is interpreted as simple string. Interesting. 

>>> '\033'
'\x1b'
>>> r'\033'
'\\033'